Code/MinimalLib/demo/demo.html

```diff
@@ -24,33 +24,87 @@
       console.log('smiles: ' + m1.get_smiles());
       m1.delete();
       callback("CC(=O)Oc1ccccc1C(=O)O");
+      //callback("CCO");
```

Comment:
If this is supposed to remain, it might be worth putting a comment in explaining what it's for.

Reply:
Yeah that was just for the phase of testing when I was popping up an alert for every line. Will delete it